proto/umee/ugov/v1/tx.proto

```diff
@@ -12,19 +12,19 @@ option (gogoproto.goproto_getters_all) = false;
 
 // Msg defines the x/ugov module's Msg service.
 service Msg {
-  // GovUpdateMinFees sets protocol controlled tx min fees.
-  rpc GovUpdateMinFees(MsgGovUpdateMinFees) returns (MsgGovUpdateMinFeesResponse);
+  // GovUpdateMinGasPrice sets protocol controlled tx min fees.
+  rpc GovUpdateMinGasPrice(MsgGovUpdateMinGasPrice) returns (MsgGovUpdateMinGasPriceResponse);
 }
 
-// MsgGovUpdateMinFees is a request type for the Msg/GovUpdateMinFees.
-message MsgGovUpdateMinFees {
+// MsgGovUpdateMinGasPrice is a request type for the Msg/GovUpdateMinGasPrice.
+message MsgGovUpdateMinGasPrice {
   option (cosmos.msg.v1.signer) = "authority";
 
   // authority must be the address of the governance account.
-  string   authority                            = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
-  repeated cosmos.base.v1beta1.Coin min_tx_fees = 2
-      [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"];
+  string                      authority     = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  cosmos.base.v1beta1.DecCoin min_gas_price = 2
+      [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.DecCoin"];
```

Comment:
I think this one also  `github.com/cosmos/cosmos-sdk/types.DecCoins` instead of `DecCoin`